config/opal_check_package.m4

```diff
@@ -134,6 +134,8 @@ AC_DEFUN([_OPAL_CHECK_PACKAGE_LIB], [
                   test "$ac_cv_search_$3" != "none required"],
                  [$1_LIBS="$ac_cv_search_$3 $4"],
                  [$1_LIBS="$4"])
+            # Check libnl version if any
```

Comment:
I think you need a better / more explanitive comment here: this is a somewhat subtle issue.  Future maintainers might (will) look at this and ask "why do we have a check for libnl in the generic package library check?"

It might also be worthwhile to rename this macro -- perhaps OPAL_LIBNL_SANITY_CHECK, or somesuch?